docs/API_Server.md

```diff
@@ -396,6 +396,22 @@ and might later produce asynchronous messages such as:
 `{"params":{"data":[[3292.432935, 562534, 0.067059278],
 [3292.4394937, 5625322, 0.670590639]]}}`
 
+### load_cell/dump_force
+
+This endpoint is used to subscribe to force data produced by a load_cell.
+Using this endpoint may increase Klipper's system load.
```

Comment:
It seems odd that there is an additional endpoint added for this information.  I would have thought the information could be added to the existing `hx71x/dump_hx71x` endpoint (or that endpoint replaced with this one).

Reply:
Well there are 2 ADC endpoints `hx71x/dump_hx71x` and `ads1220/dump_ads1220`. 

My goal with this was to try to keep everything "load cell" related out of the ADC code. The `load_cell` is acting as a decorator to the raw ADC data stream. The raw stream is still useful as a diagnostic tool (or at least it was during development).

This goes with your other question about extending/updating the socket code. What we decide here is going to determine what to do about that comment.

At a high level, what are you thinking about here?
- Is the performance penalty of the duplicate streams a concern?
- Is the duplication of the raw counts data in 2 streams confusing?
- Everything is one stream is desired but then the ADC stream is redundant/confusing?

Reply:
A bit of all three.  Mostly concerned with maintaining two different interfaces that provide redundant capabilities, and there resulting developer confusion on why there are two redundant interfaces.

Perhaps take a look at how ldc1612.py takes a calibration object and calls `self.calibration.apply_calibration(samples)`, which then allows probe_eddy_current.py to populate a "Z" field in the samples.

Reply:
I look at the `ldc1612` code and I think the other option is a better fit for this use case. The "that endpoint replaced with this one" idea.

First, I think its more developer friendly to have the endpoint be named according to the load cell in the config and not the sensor implementation. That's a leaky abstraction. That way a front end doesn't need to look up the sensor type in the config and try to build the endpoint URI. The URI can be built from the load cell name alone and its always consistent. Force data comes from a thing that is about force. That feels consistent and logical. (You could argue that LDC1212 is only about Z so that's not necessarily inconsistent)

Second but less important: the 'B' channel on the HX717 is used to measure an RTD and a hall effect filament sensor in Prusa's new machines. I think it would make supporting that kind of thing less awkward later on if we deleted the endpoint from the sensors. Objects that consume the data (LoadCell, Temperature etc.) can transform and publish it (or not) as required.

I'll work on this first.